pulsar-common/src/main/java/org/apache/pulsar/common/naming/TopicName.java

```diff
@@ -339,6 +339,42 @@ public String getPersistenceNamingEncoding() {
         }
     }
 
+    /**
+     * get topic full name from managedLedgerName.
+     *
+     * @return the topic full name, format -> domain://tenant/namespace/topic
+     */
+    public static String fromPersistenceNamingEncoding(String mlName) {
```

Comment:
Please add a unit test to cover this method.

Reply:
Unit testing has been added. PTAL @hangc0276